server/lib/mock-data.js

```diff
@@ -0,0 +1,96 @@
+const Machine = require('../classes/Machine');
+
+/**
+ * Given a machine number and a type (washer or dryer), this function creates a Machine object
+ */
+function createNewMachines(number, type) {
+	let name = type + ' ' + number;
+	let time = getMachineTime();
+	let status = getMachineStatus(time);
+	return new Machine(name, name, type, status, time);
+}
+
+/**
+ * This function spits out a random time string. It has a 1/6 chance of being available, 1/6 chance of being
+ * done running, and a 4/6 chance of being still running
+ */
+function getMachineTime() {
```

Comment:
Wait, we are serving mocks now? Or is this just for server tests? 

Would it make sense to serve mocks to the client for testing reasons? Might make life easier sometimes to not need to make clientside mocks constantly.

Reply:
After talking to jeff, it was discovered that these mocks will be served in the future as a "Sample Mode" for the app to show it off in the case that ITaP sucks and everything's offline. These exact methods were also in the old code. I blindly copied it thinking it was maybe useful to someone

Reply:
IInteresting idea. I kinda like it. Though it would be an interesting UI problem to make it clear to the user that it is just demonstration data. But I like it.